djautotask/models.py

```diff
@@ -271,3 +271,44 @@ class Project(TimeStampedModel):
 
     def __str__(self):
         return self.name
+
+
+class Task(TimeStampedModel):
+    title = models.CharField(blank=True, null=True, max_length=255)
```

Comment:
Good news, I double checked the limits on title and description and it appears they are actually telling the truth this time.